libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.scss

```diff
@@ -2,7 +2,7 @@
 @use 'libs/components/theme/src/lib/styles/variables' as *;
 
 .sky-autocomplete-results-container {
-  position: absolute;
+  position: fixed;
```

Comment:
The new issue was an affixed country field within an `overflow: hidden` element. The autocomplete changes revert back to using `position: fixed`, with the Safari on iOS issues being mitigated through other fixes.